watertap/unit_models/crystallizer.py

```diff
@@ -204,15 +205,15 @@ def build(self):
             units=pyunits.kg / pyunits.m**3,
             doc="Suspension density, i.e. density of solid-liquid mixture before separation",
         )
-
-        self.work_mechanical = Var(
-            self.flowsheet().config.time,
-            initialize=1e5,
-            bounds=(-5e6, 5e6),
-            units=pyunits.kJ / pyunits.s,
-            doc="Crystallizer thermal energy requirement",
-        )
-
+        """
+            self.work_mechanical = Var(
```

Comment:
I'm surprised to see no tests fail for the crystallizer unit especially since no changes were made to the test file.

Reply:
The tests are failing, and I'm revising them to align with the recent changes.